.github/workflows/golangci-lint.yml

```diff
@@ -22,11 +22,13 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.17
+          go-version: 1.18
       - uses: actions/checkout@v3
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
+          # golangci will install a stable go version by default, overriding the version specified above
```

Comment:
<img width="873" alt="image" src="https://user-images.githubusercontent.com/4471462/222767713-efaa1fe6-8d54-4355-acb7-e9af2cd7d4d0.png">

I think we always use the latest github action, wondering if that's an issue.

Reply:
We're using v2 which installs its own go version and still has the skip flag. I guess if we switched to golangci v3 then this goes away? But don't know if there are other changes to be aware of with that./

Reply:
Maybe worth updating yeah :)